### PR TITLE
Fixes to build on other OS

### PR DESCRIPTION
**Pull request by blondejamtart**
- change some scripts to bash interpreter
- renamed tar path variable name (old name in main entry point wasn't used by makefiles)

### CODE REVIEW

File: scripts/config.guess [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```
Comment (Araneidae) [inline scripts/config.guess]: This one script is an exception: it is a very old bit of core configuration scripting and is specifically written to use `sh` features only.  As such I'd prefer to leave this unchanged.

**Comment by Araneidae** [comment]
Could you please squash these two commits together and repush, then I can do a clean merge.  Thanks.

**Comment by thomascobb** [comment]
@Araneidae I needed this so I could test #10 as otherwise it was ignoring my setting of TAR_FILES in PandABlocks-rootfs/CONFIG

By the way, why is it TAR_DIRS in rootfs and TAR_FILES in PandABlocks-rootfs?

**Comment by Araneidae** [comment]
> By the way, why is it `TAR_DIRS` in rootfs and `TAR_FILES` in PandABlocks-rootfs?

A horrible mistake.  Looks like I got confused and muddled the names at one point.

**Pull request merged**